ktxspec.adoc

```diff
@@ -1014,6 +1014,45 @@ Still, `KHR_DF_PRIMARIES_BT709` / `KHR_DF_PRIMARIES_SRGB` is recommended
 for standard dynamic range, standard gamut images.
 ====
 
+==== Use of Transfer Functions
+The majority of transfer functions listed in Chapter 13 _Transfer
+functions_ of <<KDF13>>, with corresponding enumerators given in
+Chapter 5 _transferFunction_, have a linear opto-optical transfer
+function (OOTF). That is the opto-electrical transfer function
```

Comment:
Nitpick: the EOTF or OETF doesn't "have" an OOTF - they are defined in specifications which define an OOTF.